goty-server/build.gradle.kts

```diff
@@ -2,11 +2,10 @@ import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
 
 plugins {
     idea
-    id("org.springframework.boot") version "2.4.1"
-    id("io.spring.dependency-management") version "1.0.10.RELEASE"
-    kotlin("jvm") version "1.4.21"
-    kotlin("kapt") version "1.4.21"
```

Comment:
"dead" may be a bit of an exageration. It looked like it didn't support Java 17, but it looks like it may. Although they strongly suggest moving to KSP. What did you use kapt for?

Reply:
IIRC it has something to do with generating [metadata for Spring Boot configuration properties](https://spring.io/guides/tutorials/spring-boot-kotlin/#_configuration_properties). That being said, I'm not sure if it's needed anymore. It's hard to tell when that guide is from.

Reply:
I did some digging and [found this issue](https://github.com/spring-projects/spring-boot/issues/28046). It's not clear to me how to generate configuration property metadata with KSP. Unfortunately `annotationProcessor` by itself does not work so I removed Spring configuration processor entirely.

Since we're not publishing a Spring Boot library, it's really not important to have. The only reason for the metadata is to resolve this warning.

![image](https://user-images.githubusercontent.com/5797900/193719573-703622dc-2aad-4d33-8ab4-cb1ef590c091.png)